Sources/XCTest/Public/Asynchronous/XCTWaiter.swift

```diff
@@ -252,6 +253,42 @@ open class XCTWaiter {
         return result
     }
 
+    /// Wait on an array of expectations for up to the specified timeout, and optionally specify whether they
+    /// must be fulfilled in the given order. May return early based on fulfillment of the waited on expectations.
+    ///
+    /// - Parameter expectations: The expectations to wait on.
+    /// - Parameter timeout: The maximum total time duration to wait on all expectations.
+    /// - Parameter enforceOrder: Specifies whether the expectations must be fulfilled in the order
+    ///   they are specified in the `expectations` Array. Default is false.
+    /// - Parameter file: The file name to use in the error message if
+    ///   expectations are not fulfilled before the given timeout. Default is the file
+    ///   containing the call to this method. It is rare to provide this
+    ///   parameter when calling this method.
+    /// - Parameter line: The line number to use in the error message if the
+    ///   expectations are not fulfilled before the given timeout. Default is the line
+    ///   number of the call to this method in the calling file. It is rare to
+    ///   provide this parameter when calling this method.
+    ///
+    /// - Note: Whereas Objective-C XCTest determines the file and line
+    ///   number of the "wait" call using symbolication, this implementation
+    ///   opts to take `file` and `line` as parameters instead. As a result,
+    ///   the interface to these methods are not exactly identical between
+    ///   these environments. To ensure compatibility of tests between
+    ///   swift-corelibs-xctest and Apple XCTest, it is not recommended to pass
+    ///   explicit values for `file` and `line`.
+    @available(macOS 12.0, *) @discardableResult
```

Comment:
Can we use a single style please?  Above we use one line per attribute, here we do multiple in a single line.  A uniform style makes it easier to search through.

Reply:
Sure, I can change it.